sklearn/feature_extraction/text.py

```diff
@@ -607,9 +607,10 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         A mapping of terms to feature indices.
 
     stop_words_ : set
-        Terms that were ignored because
-        they occurred in either too many
-        (`max_df`) or in too few (`min_df`) documents.
+        Terms that were ignored because they either
+        - occurred in too many documents (`max_df`)
+        - occurred in too few documents (`min_df`)
+        - were cut off by feature selection (`max_features`).
```

Comment:
I don't think you need to prefix it with spaces, but may need a blank line before the list.